libraries/bxdf/gltf_pbr.mtlx

```diff
@@ -13,6 +13,9 @@
     <input name="alpha" type="float" value="1" uimin="0" uimax="1" uiname="Alpha" uifolder="Alpha" />
     <input name="alpha_mode" uniform="true" type="integer" enum="OPAQUE, MASK, BLEND" enumvalues="0, 1, 2" value="0" uiname="Alpha Mode" uifolder="Alpha" />
     <input name="alpha_cutoff" uniform="true" type="float" value="0.5" uimin="0" uimax="1" uiname="Alpha Cutoff" uifolder="Alpha" />
+    <input name="iridescence" type="float" value="0" uimin="0" uimax="1" uiname="Iridescence" uifolder="Iridescence" />
+    <input name="iridescence_ior" uniform="true" type="float" value="1.3" uimin="1" uisoftmax="3" uiname="Iridescence Index of Refraction" uifolder="Iridescence" />
+    <input name="iridescence_thickness" type="float" value="100" uimin="0" uisoftmin="100" uisoftmax="400" uiname="Iridescence Thickness" uifolder="Iridescence" />
```

Comment:
Hi @proog128 , 

Is the assumption here that thickness min and max needs to be handled upstream if there is a thickness texture defined as defined [here](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_iridescence/README.md).

If so then a supplementation node definition for a "gltf_iridescence_image" can be added to allow min, max and image mapping to occur. 

If not, then perhaps it would be a closer match to add in: iridescence multiplier, and and thickness min, max and ratio input ? If there is an image it would map into multiplier and ratio, and min, and max are unmapped values.

Reply:
I've added in thickness lookup as a new utility definition in the pattern PR so no need to handle it here :).